packages/generator-single-spa/src/react/templates/.babelrc

```diff
@@ -4,5 +4,9 @@
     "@babel/preset-react"
   ],
   "plugins": [
+    ["@babel/plugin-transform-runtime", {
+      "useESModules": true,
+      "regenerator": false
```

Comment:
It's arguably irresponsible to include the regenerator runtime into every bundle that needs to compile async/await or generators to ES5. It is [24kb](https://unpkg.com/browse/regenerator-runtime@0.13.3/runtime.js) that can be repeated in every webpack bundle. With the `regenerator` option turned off, it will assume that `regeneratorRuntime` is a global variable that was only loaded once. For those who don't want to deal with including regenerator runtime into their html file, they can simply change this to true.